bitnami/keycloak/templates/configmap-env-vars.yaml

```diff
@@ -35,7 +35,7 @@ data:
       {{- include "common.tplvalues.render" (dict "value" $path "context" $) }}
     {{- end }}
   {{- end }}
-  {{- if and .Values.ingress.enabled .Values.ingress.hostname }}
+  {{- if and .Values.ingress.enabled .Values.ingress.hostname (empty .Values.proxyHeaders) }}
```

Comment:
Does it apply to KEYCLOAK_HOSTNAME_ADMIN also?  https://github.com/bitnami/charts/blob/6ec5f9131660ec1735896e3a9f8885b6495955ee/bitnami/keycloak/templates/configmap-env-vars.yaml#L24-L25

Reply:
I think it's the same but it's not well documented into Keycloak documentation (https://www.keycloak.org/server/reverseproxy).
I updated the code.